package.json

```diff
@@ -43,31 +43,39 @@
     "@prisma/client": "^5.14.0",
     "@sentry/node": "^7.109.0",
     "@sentry/tracing": "^7.114.0",
+    "@shopify/shopify-api": "^11.0.0",
     "@socket.io/redis-adapter": "^8.3.0",
     "@songkeys/nestjs-redis": "^10.0.0",
     "@types/joi": "^17.2.3",
     "@ucast/sql": "^1.0.0-alpha.1",
     "class-transformer": "^0.5.1",
     "class-validator": "^0.14.1",
+    "cookie-parser": "^1.4.6",
     "express-basic-auth": "^1.2.1",
     "helmet": "^7.1.0",
+    "install": "^0.13.0",
     "ioredis": "^5.4.1",
+    "joi": "^17.13.1",
     "nodemailer-sendgrid": "^1.0.3",
+    "npm": "^10.8.1",
     "reflect-metadata": "^0.2.0",
     "rxjs": "^7.8.1",
     "socket.io": "^4.7.5",
+    "uuid": "^10.0.0",
```

Comment:
nodeJS has an internal function for UUID generation no need to use a third-party package